src/map/MapRoutePath.js

```diff
@@ -1,7 +1,9 @@
 import { useTheme } from '@mui/styles';
 import { useId, useEffect } from 'react';
 import { useSelector } from 'react-redux';
+import { featureCollection } from '@turf/turf';
```

Comment:
Can we expand `featureCollection` as well? I'm sure it's also a very simple json.